velox/functions/prestosql/registration/GeneralFunctionsRegistration.cpp

```diff
@@ -80,8 +79,6 @@ void registerAllSpecialFormGeneralFunctions() {
       bool,
       Generic<T1>,
       Variadic<Generic<T1>>>({"in"});
-  InRewrite::registerRewrite();
```

Comment:
@kagamiori, could you please remove the registration of the rewrite and retain `InRewrite.h/.cpp`? Reference: https://github.com/facebookincubator/velox/pull/15549#issuecomment-3555328606 .

Reply:
Hi @pramodsatya, sorry that I just saw your message, but this PR is already merged. Could you create another PR to add InRewrite.h/.cpp back? (By the way, we found that IN after rewrite can produce incorrect result, so it's probably safer for your use case too to fix this bug before continue using it.)

Reply:
@kagamiori, no worries, I reintroduced the IN-rewrite here: https://github.com/facebookincubator/velox/pull/15663, avoiding the rewrite for expressions of form `NULL IN [a, b, ...]` in order to respect the `kNullAsIndeterminate` null compare flag and fix the result change. Could you please take a look when you get a chance?